cli/cmd/middleware.go

```diff
@@ -35,16 +34,19 @@ An error ocurred when executing the command
 	}
 }
 
-func WithParamsHandler(params ...parameters.Params) MiddlewareWrapper {
+func WithParamsHandler(validators ...Validator) MiddlewareWrapper {
```

Comment:
this changes allows the resourcemanager client to provide its own `parameters` without needing to depend on the `parameters` package